openai/components/oai_finetune_pipeline_component.yml

```diff
@@ -0,0 +1,58 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+name: microsoft_com_azureml_oaie_finetune_pipeline_v2
+version: 0.0.1
+type: pipeline
+display_name: PipelineComponent for OpenAI Finetuning workflow
+description: Pipeline component for OpenAI finetuning workflow
+
+settings:
+  default_compute: azureml:serverless
+
+inputs:
+  model:
+    type: string
+    optional: False
+    default: DaVinci
+    description: GPT-3 model engine
+    enum:
+      - Ada
+      - Babbage
+      - Curie
+      - DaVinci
+      - Text-Ada-001
+      - Text-Babbage-001
+      - Text-Curie-001
+  train_dataset:
+    type: uri_folder
+    optional: False
+    description: Input dataset (file or folder). If a folder dataset is passed, includes all nested files.
+  registered_model_name:
+    type: string
+    optional: False
+    description: User-defined registered model name
+
+
+outputs:
+  output_model:
+    type: uri_folder
+    description: Dataset for the output model
+
+jobs:
+  model_selector:
+    type: command
+    component: azureml://registries/openai-dev-euap/components/microsoft_com_azureml_oai_modelselector/versions/0.1.7
+    inputs:
+      model: ${{parent.inputs.model}}
+
+  
+  fine_tune:
+    type: command
+    component: azureml://registries/openai-dev-euap/components/microsoft_com_azureml_oai_finetune/versions/0.1.8
```

Comment:
is this a publicly accessible registry?

Reply:
No.

Reply:
It would fail then, right?

Reply:
It would fail then, right?

Reply:
It will not, as it will be running on IPP infra.